sqlglot/dialects/tsql.py

```diff
@@ -502,6 +508,13 @@ class Parser(parser.Parser):
         STRING_ALIASES = True
         NO_PAREN_IF_COMMANDS = False
 
+        def _parse_string(self) -> t.Optional[exp.Expression]:
+            if self._curr is not None and self._curr.token_type == TokenType.NATIONAL_STRING:
+                result = self.PRIMARY_PARSERS[TokenType.NATIONAL_STRING](self, self._curr)
+                self._match(TokenType.NATIONAL_STRING)
+                return result
+            return super()._parse_string()
+
```

Comment:
Why do we need this override?

Reply:
For example, look [here](https://learn.microsoft.com/en-us/sql/t-sql/queries/hints-transact-sql-query?view=sql-server-ver16), at the `USE PLAN <XML>` syntax - it accepts regular strings as well as national strings - It might be correct in general to parse national strings as strings - WDYT?

Reply:
National strings should be parsed as [`National`](https://github.com/tobymao/sqlglot/blob/1612e622bd3514d9ca366837f47452969e5267d8/sqlglot/expressions.py#L1891) - they're not necessarily equivalent to strings, right?

Reply:
It appears that they are treated the same way (at least in the case of OPTION parsing)  - we can drop this change for the moment, and not support labels/XMLs that are passed to the OPTION clause.
WDYT?

Reply:
We should support, ideally, both of these syntaxes:
`USE PLAN N'<xml_plan>'`
`USE PLAN '<xml_plan>'`
We can check for either of them when parsing the content of the PLAN, but it appears not very generic, instead of just accepting a string IMHO..